bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,11 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% endif %}
```

Comment:
yeah, I was thinking about that. There was nothing before though so not sure what I would put in the else, unless I can do something like:

    data-outdated="{{_('vulnerable')}}"

but I very much doubt that, that will work, besides it being confusing ;) And suggestions?